core/src/wallet/cosmos/api_impl/CosmosLikeTransactionApi.cpp

```diff
@@ -345,7 +345,7 @@ namespace ledger {
 
             // Set mode
             // TODO What mode do we want? (sync|async|block)
-            vString.SetString("block", static_cast<SizeType>(5), allocator);
+            vString.SetString(mode.c_str(), static_cast<SizeType>(mode.length()), allocator);
```

Comment:
May be worth testing what happens if we broadcast with a mode other than expected values. Mode "toto" ? :)
Or just maybe sanity check and default to a known value.

Reply:
We can let the js side handle it. This way if there are changes to the protocol or the explorer we won't have to touch this part. If there's a bug in a client application, the backtrace will give the http calls so it should be easy to spot